src/Token/AccessToken.php

```diff
@@ -114,7 +114,7 @@ public function __construct(array $options = [])
                 throw new \InvalidArgumentException('expires_in value must be an integer');
             }
 
-            $this->expires = $options['expires_in'] != 0 ? $this->getTimeNow() + $options['expires_in'] : 0;
+            $this->expires = $options['expires_in'] != 0 ? $this->getTimeNow() + $options['expires_in'] : $this->getTimeNow();
```

Comment:
```suggestion
            $this->expires = $options['expires_in'] != 0
                ? $this->getTimeNow() + $options['expires_in']
                : $this->getTimeNow();
```

Fix line length style issue.